nixos/modules/services/backup/borgmatic.nix

```diff
@@ -26,17 +26,17 @@ let
     freeformType = settingsFormat.type;
     options = {
       source_directories = mkOption {
-        type = nullOr (listOf str);
-        default = null;
+        type = listOf str;
+        default = [];
         description = ''
           List of source directories and files to backup. Globs and tildes are
```

Comment:
```suggestion
          Required list of source directories and files to backup. Globs and tildes are
```

Reply:
source_directories is not required, it can be empty.

Reply:
By changing the type from `nullOr (listOf str)` to `listOf str` you made it an requirement

Reply:
But it has the default value `default = [];`.

Reply:
Ah, you're right, I missed that. Then it looks good :+1: